lib/widget/game.dart

```diff
@@ -175,6 +179,59 @@ class GameState extends State<Game> {
       ));
     }
 
+    // show the balances the accountant knows
```

Comment:
This is quite a big method. Could factor some of the widgets being created out into their own methods.